students/csdotson/lesson06/test_mailroom_4.py

```diff
@@ -0,0 +1,79 @@
+"""
+Unit testing for mailroom
+"""
+
+import mailroom_4 as mr
+import pytest
+
+def test_main_menu_prompt():
+    assert mr.main_prompt == "\nWelcome to the Main Menu! What would you like to do?\n1 - Send a Thank You\n2 - Create a Report\n3 - Send letters to everyone\nq - Quit and exit\n--> "
+
+
+def test_thank_you_prompt():
+    assert mr.thank_you_prompt == "\nPlease choose one of the following:\n1 - Add a donation and send thank you message\n2 - Display list of current donors\nq - Return to main menu\n--> "
+
+
+def test_main_menu():
+    assert mr.main_menu == {
+        "1": mr.send_thank_you,
+        "2": mr.print_report,
+        "3": mr.write_letters,
+        "q": mr.quit_menu,
+    }
+
+
+def test_thank_you_menu():
+    assert mr.thank_you_menu == {
+        "1": mr.prompt_for_donation,
+        "2": mr.print_donors,
+        "q": mr.quit_menu,
+    }
```

Comment:
I would focus on testing the parts of your program that are dynamically generated, so you probably don't need these last couple of tests since they're really just testing the initial variable assignment in your code